From 7271abf92f04db484d1eea48ab014cca7dfeac5a Mon Sep 17 00:00:00 2001
From: Lars Ellenberg <lars@linbit.com>
Date: Mon, 21 Mar 2016 13:47:12 +0100
Subject: [PATCH] fix ambiguous "GPL v 2.1"

Apparently, back in 2000, at least heartbeat/api_test.c
was first part of a "batch tag" with the LGPLv2.1 stanza,
then was manually "re-tagged" as GPLv2 (because, well,
not a library, but a test program).

But the version (2.1) was not corrected to 2 at the same time.
There is no GPL v 2.1, though.

From there this "GPLv2.1" propagated to ipfail.c,
and later attrd.c, and other source code files.

See also:
  Date: Fri, 18 Mar 2016 16:16:01 +0100
  From: Lars Ellenberg <lars.ellenberg@linbit.com>
  To: developers@clusterlabs.org
  Subject: Re: [ClusterLabs Developers] [booth][sbd] GPLv2.1+ clarification request
  Message-ID: <20160318151601.GE4437@soda.linbit>
  http://clusterlabs.org/pipermail/developers/2016-March/000180.html
  http://thread.gmane.org/gmane.comp.clustering.clusterlabs.devel/133/focus=141
  https://www.mail-archive.com/developers@clusterlabs.org/msg00137.html


Affected:
  "GPLv2.1" -> LGPL v 2.1
	lib/clplumbing/ipctransient.h
	lib/clplumbing/ipctransientclient.c
	lib/clplumbing/ipctransientlib.c
	lib/clplumbing/ipctransientserver.c
	lib/plugins/lrm/raexechb.c
	lib/plugins/lrm/raexeclsb.c
	lib/plugins/lrm/raexecocf.c
	lib/plugins/lrm/raexecupstart.c
	lib/plugins/lrm/upstart-dbus.c
	lib/plugins/lrm/upstart-dbus.h

  "GPLv2.1" -> GPL v2
	hb_report/ha_cf_support.sh
	hb_report/hb_report.in
	hb_report/openais_conf_support.sh
	hb_report/utillib.sh
	lib/clplumbing/base64_md5_test.c
	lib/clplumbing/netstring_test.c
	lrm/admin/cibsecret.in
	lrm/admin/lrmadmin.c
	lrm/test/LRMBasicSanityCheck.in
	lrm/test/evaltest.sh
	lrm/test/plugintest.c
	lrm/test/regression.sh.in

Origin: upstream, https://github.com/ClusterLabs/cluster-glue/commit/7271abf92f04db484d1eea48ab014cca7dfeac5a
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1895355
Reviewed-by: Rafael David Tinoco <rafaeldtinoco@ubuntu.com>
Last-Update: 2020-09-15
---
 hb_report/ha_cf_support.sh          | 22 +++++++++++-----------
 hb_report/hb_report.in              | 20 ++++++++++----------
 hb_report/openais_conf_support.sh   | 20 ++++++++++----------
 hb_report/utillib.sh                | 20 ++++++++++----------
 lib/clplumbing/base64_md5_test.c    | 16 ++++++++--------
 lib/clplumbing/ipctransient.h       | 14 +++++++-------
 lib/clplumbing/ipctransientclient.c | 14 +++++++-------
 lib/clplumbing/ipctransientlib.c    | 14 +++++++-------
 lib/clplumbing/ipctransientserver.c | 14 +++++++-------
 lib/clplumbing/netstring_test.c     | 22 +++++++++++-----------
 lib/plugins/lrm/raexechb.c          | 14 +++++++-------
 lib/plugins/lrm/raexeclsb.c         | 14 +++++++-------
 lib/plugins/lrm/raexecocf.c         | 14 +++++++-------
 lib/plugins/lrm/raexecupstart.c     | 14 +++++++-------
 lib/plugins/lrm/upstart-dbus.c      | 14 +++++++-------
 lib/plugins/lrm/upstart-dbus.h      | 14 +++++++-------
 lrm/admin/cibsecret.in              | 20 ++++++++++----------
 lrm/admin/lrmadmin.c                | 20 ++++++++++----------
 lrm/test/LRMBasicSanityCheck.in     | 20 ++++++++++----------
 lrm/test/evaltest.sh                | 20 ++++++++++----------
 lrm/test/plugintest.c               | 20 ++++++++++----------
 lrm/test/regression.sh.in           | 20 ++++++++++----------
 22 files changed, 190 insertions(+), 190 deletions(-)

diff --git a/hb_report/ha_cf_support.sh b/hb_report/ha_cf_support.sh
index bb63811a..3329f23a 100644
--- a/hb_report/ha_cf_support.sh
+++ b/hb_report/ha_cf_support.sh
@@ -1,17 +1,17 @@
  # Copyright (C) 2007 Dejan Muhamedagic <dmuhamedagic@suse.de>
- # 
+ #
  # This program is free software; you can redistribute it and/or
- # modify it under the terms of the GNU General Public
- # License as published by the Free Software Foundation; either
- # version 2.1 of the License, or (at your option) any later version.
- # 
- # This software is distributed in the hope that it will be useful,
+ # modify it under the terms of the GNU General Public License
+ # as published by the Free Software Foundation; either version 2
+ # of the License, or (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
  # but WITHOUT ANY WARRANTY; without even the implied warranty of
- # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- # General Public License for more details.
- # 
- # You should have received a copy of the GNU General Public
- # License along with this library; if not, see <http://www.gnu.org/licenses/>
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this library; if not, see <http://www.gnu.org/licenses/>
  #
 
 #
diff --git a/hb_report/hb_report.in b/hb_report/hb_report.in
index cc67303e..b0a96184 100755
--- a/hb_report/hb_report.in
+++ b/hb_report/hb_report.in
@@ -3,17 +3,17 @@
  # Copyright (C) 2007 Dejan Muhamedagic <dmuhamedagic@suse.de>
  # 
  # This program is free software; you can redistribute it and/or
- # modify it under the terms of the GNU General Public
- # License as published by the Free Software Foundation; either
- # version 2.1 of the License, or (at your option) any later version.
- # 
- # This software is distributed in the hope that it will be useful,
+ # modify it under the terms of the GNU General Public License
+ # as published by the Free Software Foundation; either version 2
+ # of the License, or (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
  # but WITHOUT ANY WARRANTY; without even the implied warranty of
- # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- # General Public License for more details.
- # 
- # You should have received a copy of the GNU General Public
- # License along with this library; if not, see <http://www.gnu.org/licenses/>
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this library; if not, see <http://www.gnu.org/licenses/>
  #
 
 . @OCF_ROOT_DIR@/lib/heartbeat/ocf-shellfuncs
diff --git a/hb_report/openais_conf_support.sh b/hb_report/openais_conf_support.sh
index 0a28638e..f61c200c 100644
--- a/hb_report/openais_conf_support.sh
+++ b/hb_report/openais_conf_support.sh
@@ -1,17 +1,17 @@
  # Copyright (C) 2007 Dejan Muhamedagic <dmuhamedagic@suse.de>
  # 
  # This program is free software; you can redistribute it and/or
- # modify it under the terms of the GNU General Public
- # License as published by the Free Software Foundation; either
- # version 2.1 of the License, or (at your option) any later version.
- # 
- # This software is distributed in the hope that it will be useful,
+ # modify it under the terms of the GNU General Public License
+ # as published by the Free Software Foundation; either version 2
+ # of the License, or (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
  # but WITHOUT ANY WARRANTY; without even the implied warranty of
- # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- # General Public License for more details.
- # 
- # You should have received a copy of the GNU General Public
- # License along with this library; if not, see <http://www.gnu.org/licenses/>
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this library; if not, see <http://www.gnu.org/licenses/>
  #
 
 #
diff --git a/hb_report/utillib.sh b/hb_report/utillib.sh
index 5c854abe..59727888 100644
--- a/hb_report/utillib.sh
+++ b/hb_report/utillib.sh
@@ -1,17 +1,17 @@
  # Copyright (C) 2007 Dejan Muhamedagic <dmuhamedagic@suse.de>
  # 
  # This program is free software; you can redistribute it and/or
- # modify it under the terms of the GNU General Public
- # License as published by the Free Software Foundation; either
- # version 2.1 of the License, or (at your option) any later version.
- # 
- # This software is distributed in the hope that it will be useful,
+ # modify it under the terms of the GNU General Public License
+ # as published by the Free Software Foundation; either version 2
+ # of the License, or (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
  # but WITHOUT ANY WARRANTY; without even the implied warranty of
- # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- # General Public License for more details.
- # 
- # You should have received a copy of the GNU General Public
- # License along with this library; if not, see <http://www.gnu.org/licenses/>
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this library; if not, see <http://www.gnu.org/licenses/>
  #
 
 #
diff --git a/lib/clplumbing/base64_md5_test.c b/lib/clplumbing/base64_md5_test.c
index 969b7f39..67ede7cb 100644
--- a/lib/clplumbing/base64_md5_test.c
+++ b/lib/clplumbing/base64_md5_test.c
@@ -5,17 +5,17 @@
  * Copyright (c) 2005 International Business Machines
  *
  * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
  *
- * This software is distributed in the hope that it will be useful,
+ * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU General Public
- * License along with this library; if not, see <http://www.gnu.org/licenses/>
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>
  */
 
 #include <lha_internal.h>
diff --git a/lib/clplumbing/ipctransient.h b/lib/clplumbing/ipctransient.h
index 0c90fbbf..7844050e 100644
--- a/lib/clplumbing/ipctransient.h
+++ b/lib/clplumbing/ipctransient.h
@@ -1,17 +1,17 @@
 /* 
  * Copyright (C) 2007 Andrew Beekhof <andrew@beekhof.net>
  * 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  */
 #undef _GNU_SOURCE  /* in case it was defined on the command line */
diff --git a/lib/clplumbing/ipctransientclient.c b/lib/clplumbing/ipctransientclient.c
index f169573b..72d19639 100644
--- a/lib/clplumbing/ipctransientclient.c
+++ b/lib/clplumbing/ipctransientclient.c
@@ -1,17 +1,17 @@
 /* 
  * Copyright (C) 2004 Andrew Beekhof <andrew@beekhof.net>
  * 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  */
 
diff --git a/lib/clplumbing/ipctransientlib.c b/lib/clplumbing/ipctransientlib.c
index bdd5c28b..7bea6889 100644
--- a/lib/clplumbing/ipctransientlib.c
+++ b/lib/clplumbing/ipctransientlib.c
@@ -1,17 +1,17 @@
 /* 
  * Copyright (C) 2004 Andrew Beekhof <andrew@beekhof.net>
  * 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  */
 
diff --git a/lib/clplumbing/ipctransientserver.c b/lib/clplumbing/ipctransientserver.c
index 58440b07..9920d9fd 100644
--- a/lib/clplumbing/ipctransientserver.c
+++ b/lib/clplumbing/ipctransientserver.c
@@ -1,17 +1,17 @@
 /* 
  * Copyright (C) 2004 Andrew Beekhof <andrew@beekhof.net>
  * 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  */
 
diff --git a/lib/clplumbing/netstring_test.c b/lib/clplumbing/netstring_test.c
index 63d46e4f..f92551d0 100644
--- a/lib/clplumbing/netstring_test.c
+++ b/lib/clplumbing/netstring_test.c
@@ -3,18 +3,18 @@
  *
  * Copyright (C) 2000 Guochun Shi <gshi@ncsa.uiuc.edu>
  * 
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
- * License along with this library; if not, see <http://www.gnu.org/licenses/>
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>
  */
 
 #include <lha_internal.h>
diff --git a/lib/plugins/lrm/raexechb.c b/lib/plugins/lrm/raexechb.c
index ac22c6c2..32c91ab2 100644
--- a/lib/plugins/lrm/raexechb.c
+++ b/lib/plugins/lrm/raexechb.c
@@ -1,15 +1,15 @@
 /* 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  *
  * File: raexechb.c
diff --git a/lib/plugins/lrm/raexeclsb.c b/lib/plugins/lrm/raexeclsb.c
index 370e5d30..db9f6e63 100644
--- a/lib/plugins/lrm/raexeclsb.c
+++ b/lib/plugins/lrm/raexeclsb.c
@@ -1,15 +1,15 @@
 /* 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  *
  * File: raexeclsb.c
diff --git a/lib/plugins/lrm/raexecocf.c b/lib/plugins/lrm/raexecocf.c
index 0e06d304..0e32c7fd 100644
--- a/lib/plugins/lrm/raexecocf.c
+++ b/lib/plugins/lrm/raexecocf.c
@@ -1,15 +1,15 @@
 /* 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  *
  * File: raexecocf.c
diff --git a/lib/plugins/lrm/raexecupstart.c b/lib/plugins/lrm/raexecupstart.c
index e38af205..05050520 100644
--- a/lib/plugins/lrm/raexecupstart.c
+++ b/lib/plugins/lrm/raexecupstart.c
@@ -1,15 +1,15 @@
 /* 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  *
  * File: raexecupstart.c
diff --git a/lib/plugins/lrm/upstart-dbus.c b/lib/plugins/lrm/upstart-dbus.c
index 8c79726c..45a2256f 100644
--- a/lib/plugins/lrm/upstart-dbus.c
+++ b/lib/plugins/lrm/upstart-dbus.c
@@ -1,15 +1,15 @@
 /* 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  *
  * File: upstart-dbus.c
diff --git a/lib/plugins/lrm/upstart-dbus.h b/lib/plugins/lrm/upstart-dbus.h
index 91d16729..bd981938 100644
--- a/lib/plugins/lrm/upstart-dbus.h
+++ b/lib/plugins/lrm/upstart-dbus.h
@@ -1,15 +1,15 @@
 /* 
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
  * License as published by the Free Software Foundation; either
  * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ *
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
  * License along with this library; if not, see <http://www.gnu.org/licenses/>
  *
  * File: upstart-dbus.c
diff --git a/lrm/admin/cibsecret.in b/lrm/admin/cibsecret.in
index 166be087..5278a378 100755
--- a/lrm/admin/cibsecret.in
+++ b/lrm/admin/cibsecret.in
@@ -3,17 +3,17 @@
 # Copyright (C) 2011 Dejan Muhamedagic <dmuhamedagic@suse.de>
 # 
 # This program is free software; you can redistribute it and/or
-# modify it under the terms of the GNU General Public
-# License as published by the Free Software Foundation; either
-# version 2.1 of the License, or (at your option) any later version.
-# 
-# This software is distributed in the hope that it will be useful,
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-# General Public License for more details.
-# 
-# You should have received a copy of the GNU General Public
-# License along with this library; if not, see <http://www.gnu.org/licenses/>
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this library; if not, see <http://www.gnu.org/licenses/>
 #
 
 # WARNING:
diff --git a/lrm/admin/lrmadmin.c b/lrm/admin/lrmadmin.c
index b51f287c..0cf504f2 100644
--- a/lrm/admin/lrmadmin.c
+++ b/lrm/admin/lrmadmin.c
@@ -7,17 +7,17 @@
  * Todo: security verification
  *
  * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
- * License along with this library; if not, see <http://www.gnu.org/licenses/>
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>
  */
 #include <lha_internal.h>
 
diff --git a/lrm/test/LRMBasicSanityCheck.in b/lrm/test/LRMBasicSanityCheck.in
index 801b295d..5d80359b 100755
--- a/lrm/test/LRMBasicSanityCheck.in
+++ b/lrm/test/LRMBasicSanityCheck.in
@@ -4,17 +4,17 @@
  # Author: Huang Zhen <zhenhltc@cn.ibm.com>
  # 
  # This program is free software; you can redistribute it and/or
- # modify it under the terms of the GNU General Public
- # License as published by the Free Software Foundation; either
- # version 2.1 of the License, or (at your option) any later version.
- # 
- # This software is distributed in the hope that it will be useful,
+ # modify it under the terms of the GNU General Public License
+ # as published by the Free Software Foundation; either version 2
+ # of the License, or (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
  # but WITHOUT ANY WARRANTY; without even the implied warranty of
- # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- # General Public License for more details.
- # 
- # You should have received a copy of the GNU General Public
- # License along with this library; if not, see <http://www.gnu.org/licenses/>
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this library; if not, see <http://www.gnu.org/licenses/>
  #
 HBLIB=@libdir@/heartbeat
 LRMD=$HBLIB/lrmd
diff --git a/lrm/test/evaltest.sh b/lrm/test/evaltest.sh
index 93ae4835..4bbd52b0 100755
--- a/lrm/test/evaltest.sh
+++ b/lrm/test/evaltest.sh
@@ -3,17 +3,17 @@
  # Copyright (C) 2007 Dejan Muhamedagic <dejan@suse.de>
  # 
  # This program is free software; you can redistribute it and/or
- # modify it under the terms of the GNU General Public
- # License as published by the Free Software Foundation; either
- # version 2.1 of the License, or (at your option) any later version.
- # 
- # This software is distributed in the hope that it will be useful,
+ # modify it under the terms of the GNU General Public License
+ # as published by the Free Software Foundation; either version 2
+ # of the License, or (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
  # but WITHOUT ANY WARRANTY; without even the implied warranty of
- # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- # General Public License for more details.
- # 
- # You should have received a copy of the GNU General Public
- # License along with this library; if not, see <http://www.gnu.org/licenses/>
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this library; if not, see <http://www.gnu.org/licenses/>
  #
 
 : ${TESTDIR:=testcases}
diff --git a/lrm/test/plugintest.c b/lrm/test/plugintest.c
index e9efa614..bc5e2bd5 100644
--- a/lrm/test/plugintest.c
+++ b/lrm/test/plugintest.c
@@ -7,17 +7,17 @@
  * Todo: security verification
  *
  * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- * 
- * This software is distributed in the hope that it will be useful,
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
- * License along with this library; if not, see <http://www.gnu.org/licenses/>
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>
  */
 #include <glib.h>
 #include <stdio.h>
diff --git a/lrm/test/regression.sh.in b/lrm/test/regression.sh.in
index edab02e6..cec692d9 100755
--- a/lrm/test/regression.sh.in
+++ b/lrm/test/regression.sh.in
@@ -3,17 +3,17 @@
  # Copyright (C) 2007 Dejan Muhamedagic <dmuhamedagic@suse.de>
  # 
  # This program is free software; you can redistribute it and/or
- # modify it under the terms of the GNU General Public
- # License as published by the Free Software Foundation; either
- # version 2.1 of the License, or (at your option) any later version.
- # 
- # This software is distributed in the hope that it will be useful,
+ # modify it under the terms of the GNU General Public License
+ # as published by the Free Software Foundation; either version 2
+ # of the License, or (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
  # but WITHOUT ANY WARRANTY; without even the implied warranty of
- # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- # General Public License for more details.
- # 
- # You should have received a copy of the GNU General Public
- # License along with this library; if not, see <http://www.gnu.org/licenses/>
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this library; if not, see <http://www.gnu.org/licenses/>
  #
 
 OCF_ROOT=@OCF_ROOT_DIR@
-- 
2.27.0

